vtexdocs/devportal
# Feat/faststore next mdx

#### What is the purpose of this pull request?

To add support to the FastStore documentations.

#### What problem is this solving?

The developers portal was unable to show FastStore docs.

#### How should this be manually tested?

Open the preview, try to open any FastStore doc and check if the documentation is correct, try to also open the other devportal documentations to verify if they are still being displayed correctly. The sidebar should not show the FastStore documentation.

Examples:
[Button](https://deploy-preview-433--elated-hoover-5c29bf.netlify.app/docs/guides/faststore/atoms-button)
[Regionalization](https://deploy-preview-433--elated-hoover-5c29bf.netlify.app/docs/guides/faststore/features-regionalization)
[Go Live: setting up VTEX account for FastStore](https://deploy-preview-433--elated-hoover-5c29bf.netlify.app/docs/guides/faststore/go-live-1-setting-up-vtex-account-for-faststore)

#### Screenshots or example usage

![image](https://github.com/vtexdocs/devportal/assets/62757720/6688017e-e0f3-4fff-aed5-12f4f1df7104)

#### Types of changes
